src/main/java/org/dependencytrack/resources/v1/TeamResource.java

```diff
@@ -220,26 +192,77 @@ public Response deleteTeam(Team jsonTeam) {
         }
     }
 
+    @GET
+    @Path("/available-teams")
```

Comment:
Consider renaming to `/visible` instead, since this endpoint returns teams *visible* to the authenticated user.